native/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -0,0 +1,147 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.persist.cmd;
+
+import io.ballerina.cli.BLauncherCmd;
+import io.ballerina.persist.PersistToolsConstants;
+import io.ballerina.persist.nodegenerator.CreateSyntaxTree;
+import io.ballerina.projects.Project;
+import io.ballerina.projects.ProjectEnvironmentBuilder;
+import io.ballerina.projects.ProjectException;
+import io.ballerina.projects.directory.ProjectLoader;
+import io.ballerina.toml.syntax.tree.SyntaxTree;
+import org.ballerinalang.formatter.core.Formatter;
+import org.ballerinalang.formatter.core.FormatterException;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.io.PrintWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Class to implement "persist init" command for ballerina.
+ */
+
+@CommandLine.Command(
+        name = "init",
+        description = "generate database configurations.")
+
+public class Init implements BLauncherCmd {
+    private final PrintStream outStream = System.out;
+    private final PrintStream errStream = System.err;
+    private String sourcePath = "";
+    private ProjectEnvironmentBuilder projectEnvironmentBuilder;
+    private String configPath = PersistToolsConstants.CONFIG_PATH;
```

Comment:
```suggestion
    private final String configPath = PersistToolsConstants.CONFIG_PATH;
```

Reply:
Updated in [8078951](https://github.com/ballerina-platform/persist-tools/pull/9/commits/8078951dff409765a286bc5e8f289d15b9b412ee)